docs/contributorfile.rst

```diff
@@ -132,75 +156,144 @@ Issues labels
+- The PR has the appropriate labels to trigger the appropriate version release and update the contributors table.
+
+.. _mainreviewer:
+
+Main reviewer
+~~~~~~~~~~~~~
+At ``physiopy`` we use the "Assignees" section of a PR to mark the **main reviewer** for that PR. The main reviewer is the main responsible **for the quality of the repository and its next version release**, as well as **for the behaviour of the other reviewers**. 
+The main reviewer:
+
+- Takes care of the reviewing process of the PR, in particular:
+    - Invites the reviewers to finish their review in a relatively short time.
+    - Checks that this document was respected in all its part, especially the part about `Pull Requests <#pr>`_.
+    - Invites other Reviewers to respect this document, especially the part about `reviews <#reviewing>`_, helps them in doing so, and checks that they do.
+    - If a Reviewer keeps not respecting this document, flags them to the project manager.
+- Decides what to do in case of a coverage decrease (in *codecov/patch*).
+- In case of missing tests or updates to user documentation:
+    - Asks for more documentation or tests before approving the PR, *or*
+    - Checks that the adequate issues have been opened to address the lack of documentation or tests (1 issue per item).
+- Double-checks that the title is clear and the labels are correct to trigger an adequate ``auto`` release - feel free to change them.
+- **Is the one that is going to merge the PR.**
+- After the PR got merged and a new release was triggered, checks that:
+    - The documentation was updated correctly (if changed).
+    - The Pypi version of the repository coincides with the new release (if changed).
+    - New contributors or forms of contributions were correctly added in the README (if changed).
 
 .. _styling:
 
 Style Guide
 -----------
 Docstrings should follow `numpydoc <https://numpydoc.readthedocs.io/en/latest/format.html>`_ convention. We encourage extensive documentation.
 The python code itself should follow `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ convention whenever possible: there are continuous integration tests checking that!
-You can use linters to help you write your code following style conventions. Linters are add-ons that you can run on the written script file. We suggest the use of **flake8** for Python 3. Many editors (Atoms, VScode, Sublimetext, ...) support addons for online lintering, which means you’ll see warnings and errors while you write the code - check out if your does!
+You can use linters to help you write your code following style conventions. Linters are add-ons that you can run on the written script file. We suggest the use of **flake8** for Python 3. Many editors (Atom, VScode, Sublimetext, ...) support addons for online lintering, which means you’ll see warnings and errors while you write the code - check out if your does!
```

Comment:
Should we wait for the discussion going on at #324 before merging?

Reply:
Even if we adopt an opinionated code formatter, I wouldn't force the contributors to adopt it during their coding or documentation writing. In fact, one of the reasons to adopt an opinionated code formatter is to _not have to respect any opinionated code format during coding_. Most of the time it's sufficient that the code respects *flake8* conventions, so that the changes introduced by opinionated code formatters do not introduce an incredible amount of differences compared to the submitted code - although it's true that it becomes less essential. Since we're going to have `black` run during PR merge (I guess?), I would keep this section as it is.